ivanti_security_controls/plugin.spec.yaml

```diff
@@ -562,3 +738,62 @@ actions:
         description: Detected patches
         type: '[]detected_patch'
         required: true
+  search_patches:
+    title: Search Patches
+    description: Find and display detailed information about patch
+    input:
+      security_id:
+        title: Security ID
+        description: Security Vulnerability ID
+        type: '[]string'
+        required: true
+        example: ["MS99-031", "Q240346", "CVE-2015-4485", "4693"]
```

Comment:
Does Ivanti require upper-case for letters?

Can the user pass in `["ms99-031", "cve-2015-4485" ]` and will it work?

If not, can we call `.upper()` so the user can pass `cve-..` or `CVE-..`

Reply:
I check it and  fix it.